mediaroute/build.gradle

```diff
@@ -0,0 +1,121 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-kapt'
+
+configurations {
+    ktlint
+}
+
+androidExtensions {
+    experimental = true
+}
+
+buildscript {
+    ext {
+        junit_version = '4.12'
+        kotlin_version = '1.3.31'
+        support_lib_version = '28.0.0'
+        ktlint_version = '0.31.0'
+        junit_version = '4.12'
+        mockito_core_version = '2.27.0'
+        mockito_kotlin_version = '2.1.0'
+        powermock_version = '2.0.2'
+        harmcrest_version = '1.3'
+    }
+}
+
+android {
+    compileSdkVersion 28
+
+    defaultConfig {
+        minSdkVersion 19
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+        multiDexEnabled true
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+
+    compileOptions {
+        encoding 'UTF-8'
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+        debug {
+            minifyEnabled false
+        }
+    }
+
+    dataBinding {
+        enabled = true
+    }
+}
+
+dependencies {
+    ktlint "com.github.shyiko:ktlint:$ktlint_version"
+
+    implementation 'com.android.support:multidex:1.0.3'
+
+    // OCast SDK v2
+    implementation project(':OCast')
+
+    // Kotlin
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    // Support libraries
+    implementation "com.android.support:support-v4:$support_lib_version"
+    implementation "com.android.support:appcompat-v7:$support_lib_version"
+    implementation "com.android.support:mediarouter-v7:$support_lib_version"
+
+    def withoutHamcrest = {
```

Comment:
Remove this line:
`androidTestImplementation "org.hamcrest:hamcrest-all:$harmcrest_version"`
instead of excluding Harmcrest from `runner` and `espresso`.